server/helpers/sendTelegram.js

```diff
@@ -51,24 +51,29 @@ ${welcomeText}`,
 }
 
 const getTelegramBody = async subscriber => {
-  let delegatorAccount, transcoderAccount, currentRound
+  let delegatorAccount, transcoderAccount, currentRoundObject
   await promiseRetry(async retry => {
     // Get delegator Account
     try {
       delegatorAccount = await getLivepeerDelegatorAccount(subscriber.address)
       if (delegatorAccount && delegatorAccount.status == 'Bonded') {
         // Get transcoder account
         transcoderAccount = await getLivepeerTranscoderAccount(delegatorAccount.delegateAddress)
-        currentRound = await getLivepeerCurrentRound()
+        currentRoundObject = await getLivepeerCurrentRoundInfo()
       }
     } catch (err) {
       retry()
     }
   })
-  if (!delegatorAccount || !transcoderAccount || !currentRound) {
-    throw new Error('There is no telegram to send')
+  console.log(`Delegator account ${JSON.stringify(delegatorAccount)}`)
```

Comment:
Could you delete this or remplace it with a logger?, thanks

Reply:
@Agupane This log is for control purposes, Papertrail will register it and we can use it to verify possible problems. Logger is not used in the backend, it is more to use with chrome dev tools. Thanks for the advice, but it is recommended to keep like this.
Regards